plugins/defaultplugins/ifplugin/data_resync.go

```diff
@@ -676,3 +600,143 @@ func (plugin *NatConfigurator) resolveMappings(nbDNatConfig *nat.Nat44DNat_DNatC
 		}
 	}
 }
+
+// Compares two interfaces. If there is any difference, returns true, false otherwise
+func (plugin *InterfaceConfigurator) isIfModified(nbIf, vppIf *intf.Interfaces_Interface) (isModified bool) {
+	isModified = true
```

Comment:
Using variable for this seems completely unnecessary and less readable in return calls. Just return the boolean value directly.